.github/workflows/_move_edd_db_scripts.yml

```diff
@@ -61,35 +62,58 @@ jobs:
         id: branch_name
         env:
           PREFIX: ${{ needs.setup.outputs.migration_filename_prefix }}
-        run: echo "branch_name=move_finalization_db_scripts_$PREFIX" >> $GITHUB_OUTPUT
+        run: echo "branch_name=move_edd_db_scripts_$PREFIX" >> $GITHUB_OUTPUT
 
       - name: "Create branch"
         env:
           BRANCH: ${{ steps.branch_name.outputs.branch_name }}
         run: git switch -c $BRANCH
 
-      - name: Move DbScripts_finalization
+      - name: Move scripts and finalization database schema
         id: move-files
         env:
           PREFIX: ${{ needs.setup.outputs.migration_filename_prefix }}
         run: |
-          src_dir="util/Migrator/DbScripts_finalization"
+          moved_files=""
+
+          # scripts
+          src_dirs="util/Migrator/DbScripts_transition,util/Migrator/DbScripts_finalization"
```

Comment:
ℹ️ Gathers and moves all the scripts with the existing renaming. Transition scripts happen before finalization.